src/views/accounts/const.js

```diff
@@ -139,7 +143,7 @@ export const gatherAccountHeaderActions = (vm) => {
           })
           vm.$axios.put(
             `/api/v1/accounts/gathered-accounts/status/`,
-            { ids: ids, status: 'confirmed' }
+            { ids: ids, status: statusMap.confirmed }
           ).then(() => {
             vm.$message.success(vm.$tc('SyncSuccessMsg'))
           }).catch(err => {
```

Comment:
I recommend reviewing these parts of the code:

  * Make sure you have up to date time utilities that work with JavaScript's Date Object.
  * Use proper formatting string templates instead of directly calling `toSafeLocalDateStr` for consistency and readability.
  * Consider using const to avoid unnecessary function redeclaration.

Let me know if this helps!